diff --git a/node_modules/send-intent/android/build.gradle b/node_modules/send-intent/android/build.gradle
index 78bb52f..efb8039 100644
--- a/node_modules/send-intent/android/build.gradle
+++ b/node_modules/send-intent/android/build.gradle
@@ -21,11 +21,11 @@ apply plugin: 'com.android.library'
 
 android {
     namespace "de.mindlib.sendIntent.sendintent"
-    compileSdkVersion 35
+    compileSdkVersion 36
 
     defaultConfig {
         minSdkVersion 23
-        targetSdkVersion 35
+        targetSdkVersion 36
         versionCode 1
         versionName "1.0"
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
